mbf_abstract_nav/CMakeLists.txt

```diff
@@ -16,6 +16,8 @@ find_package(catkin REQUIRED
   std_msgs
   std_srvs
   tf
+  visualization_msgs
```

Comment:
Yes there is a visualization_msgs::Marker. But I think it is unused. So we should remove the include.

Reply:
agree